content/messaging/customer-chat-api/v3.2/index.mdx

```diff
@@ -2202,21 +2211,21 @@ curl -X POST \
 
 <Text>
 
-### Set Customer Fields
+### Set Customer Session Fields
 
 #### Specifics
 
-|                        |                                                                        |
-| ---------------------- | ---------------------------------------------------------------------- |
-| **Method URL**         | `https://api.livechatinc.com/v3.2/customer/action/set_customer_fields` |
-| **RTM API equivalent** | [`set_customer_fields`](rtm-reference/#set-customer-fields)            |
-| **Webhook**            | [`customer_updated`](/management/configuration-api/#customer_updated)  |
+|                        |                                                                                |
+| ---------------------- | ------------------------------------------------------------------------------ |
+| **Method URL**         | `https://api.livechatinc.com/v3.2/customer/action/set_customer_session_fields` |
+| **RTM API equivalent** | [`set_customer_session_fields`](rtm-reference/#set-customer-session-fields)    |
+| **Webhook**            | [`customer_updated`](/management/configuration-api/#customer_updated)          |
 
 #### Request
 
-| Parameter | Required | Data type | Notes                              |
-| --------- | -------- | --------- | ---------------------------------- |
-| `fields`  | Yes      | `object`  | `key:value` format; max keys: 100  |
+| Parameter        | Required | Data type  | Notes                                                                                            |
+| ---------------- | -------- | ---------- | ------------------------------------------------------------------------------------------------ |
+| `session_fields` | No       | `[]object` | An array of custom object-enclosed `key:value` pairs. Respects the order of items. Max keys: 100 |
```

Comment:
not required?

Reply:
https://github.com/livechat/customer-api/blob/master/parsers/3.2/methods/set-customer-session-fields.go#L118

Reply:
I see that this copy-paste error rattled you 😉